public/assets/css/styles.css

```diff
@@ -0,0 +1,46 @@
+@import url(./main.css);
```

Comment:
Can you move all the contents of this file back to `main.css` and remove this one? Please also move the `/public/css` folder back to the root and remove the `/public` folder.